### DISCUSSION TITLE
How to  integrate opnForm into our custom Vue3 application?

### DISCUSSION THREAD

**Discussion by krupa5** [OpnForm/OpnForm#86] [Q&A] [answered]
Hi,

We are trying to integrate opnForm into our custom Vue3 application. Kindly provide guidance on the steps necessary for integrating the form code into our vue3 setup in a thorough and efficient manner. This will help us a lot to get started. As a standalone, we have already executed and it seems to be working fine, just need to integrate within our project.

**Comment by JhumanJ** [comment, marked as answer]
Hi, OpnForm is a standalone project and it cannot be integrated in another application (+ it's a Vue2 project).
